pkg/services/postgresql/all_in_one_types.go

```diff
@@ -31,41 +27,41 @@ func (
 func (
 	a *allInOneManager,
 ) GetEmptySecureProvisioningParameters() service.SecureProvisioningParameters {
-	return &SecureAllInOneProvisioningParameters{}
+	return nil
```

Comment:
Another pervasive change: these sort of functions can now return `nil` if they'd otherwise be returning a useless struct type that has no fields.

Reply:
:+1: